src/main/resources/application-dev.yml

```diff
@@ -1,3 +1,9 @@
 spring:
   flyway:
     locations: classpath:/migrations/common,classpath:/migrations/dev
+
+notify:
```

Comment:
This is the new template with the additional place holder fields included in DEV only.

In all other environments it will carry on using the old email which does not expect the new placeholder values.

Reply:
Would you be able to add these instructions in the rollout confluence page please?